data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Саймонс", "Simons" ],
+      [ "Сальдивар", "Saldivar" ],
+      [ "Салмон", "Salmon" ],
+      [ "Самора", "Zamora" ],
+      [ "Сэмюэл", "Samuel" ],
+      [ "Бонд", "Bond" ],
+      [ "Сэлф", "Self" ],
+      [ "Сэвидж", "Savage" ],
+      [ "Норт", "North" ],
+      [ "Сент-Джон", "St. John" ],
+      [ "Сильвер", "Silver" ],
+      [ "Сильвестр", "Sylvester" ],
+      [ "Сильвия", "Silvia" ],
+      [ "Стронг", "Strong" ],
+      [ "Синглтэри", "Singletary" ],
+      [ "Стэйплс", "Staples" ],
+      [ "Слэк", "Slack" ],
+      [ "Слотэр", "Slaughter" ],
+      [ "Слейтер", "Slater" ],
+      [ "Хёрд", "Heard" ],
+      [ "Смоллвуд", "Smallwood" ],
+      [ "Соарес", "Soares" ],
+      [ "Спэйн", "Spain" ],
+      [ "Спрэйг", "Sprague" ],
+      [ "Старнс", "Starnes" ],
+      [ "Стейли", "Staley" ],
+      [ "Стейнберг", "Steinberg" ],
+      [ "Стэйтон", "Staton" ],
+      [ "Гласс", "Glass" ],
+      [ "Уолл", "Wall" ],
+      [ "Сент-Клер", "Stclair" ],
+      [ "Стовалл", "Stovall" ],
+      [ "Стовер", "Stover" ],
+      [ "Столлингс", "Stallings" ],
+      [ "Стоукс", "Stokes" ],
+      [ "Пейдж", "Page" ],
+      [ "Стрейндж", "Strange" ],
+      [ "Стрэттон", "Stratton" ],
+      [ "Стэнфилд", "Stanfield" ],
+      [ "Сунига", "Zuniga" ],
+      [ "Сэй", "Seay" ],
+      [ "Сэйлор", "Saylor" ],
+      [ "Таварес", "Tavares" ],
+      [ "Таннер", "Tanner" ],
+      [ "Тейшейра", "Teixeira" ],
+      [ "Теллез", "Tellez" ],
+      [ "Тернер", "Turner" ],
+      [ "Титус", "Titus" ],
+      [ "Толберт", "Tolbert" ],
+      [ "Мерчант", "Merchant" ],
+      [ "Траммелл", "Trammell" ],
+      [ "Траутман", "Troutman" ],
+      [ "Труонг", "Truong" ],
+      [ "Трэхан", "Trahan" ],
+      [ "Тэкетт", "Tackett" ],
+      [ "Тэм", "Tam" ],
+      [ "Бэйлс", "Bales" ],
+      [ "Тёрпин", "Turpin" ],
+      [ "Уайлс", "Wiles" ],
+      [ "Уиллер", "Wheeler" ],
+      [ "Уиллингем", "Willingham" ],
+      [ "Уиллоби", "Willoughby" ],
+      [ "Уинн", "Winn" ],
+      [ "Уиттен", "Whitten" ],
+      [ "Уиттингтон", "Whittington" ],
+      [ "Смарт", "Smart" ],
+      [ "Уоркмен", "Workman" ],
+      [ "Уотерман", "Waterman" ],
+      [ "Уэбстер", "Webster" ],
+      [ "Уэллетт", "Ouellette" ],
+      [ "Уэлч", "Welch" ],
+      [ "Фалкон", "Falcon" ],
+      [ "Фаррис", "Farris" ],
+      [ "Фейр", "Fair" ],
+      [ "Фитцпатрик", "Fitzpatrick" ],
+      [ "Флеминг", "Fleming" ],
+      [ "Фой", "Foy" ],
+      [ "Фоли", "Foley" ],
+      [ "Фоллон", "Fallon" ],
+      [ "Фомэн", "Forman" ],
+      [ "Фонтено", "Fontenot" ],
+      [ "Траут", "Trout" ],
+      [ "Франк", "Frank" ],
+      [ "Франкс", "Franks" ],
+      [ "Франс", "France" ],
+      [ "Фрейли", "Fraley" ],
+      [ "Фрейтас", "Freitas" ],
+      [ "Фримен", "Freeman" ],
+      [ "Фуртадо", "Furtado" ],
+      [ "Фэган", "Fagan" ],
+      [ "Хадсон", "Hudson" ],
+      [ "Хайтауер", "Hightower" ],
+      [ "Хакетт", "Hackett" ],
+      [ "Хамель", "Hamel" ],
```

Comment:
Should be [`Хэмел`](https://ru.wikipedia.org/wiki/%D0%A5%D1%8D%D0%BC%D0%B5%D0%BB). Worth noting that this could also be a French and German name, where it would be `Амель` and `Гамель`, respectively.